04-Enumerable/Enumerable/Task.cs

```diff
@@ -322,13 +362,19 @@ public class Task {
         ///    {(1/1/2010, 10)  , (2/2/2010, 10), (3/3/2010, 10) } => { 30, 0, 0, 0 }
         ///    {(1/1/2010, 10)  , (4/4/2010, 10), (10/10/2010, 10) } => { 10, 10, 0, 10 }
         /// </example>
-        public int[] GetQuarterSales(IEnumerable<Tuple<DateTime, int>> sales) {
+        public int[] GetQuarterSales(IEnumerable<Tuple<DateTime, int>> sales)
+        {
             // TODO : Implement GetQuarterSales
-            throw new NotImplementedException();
+            var firstQuarter = sales.Where(x => x.Item1.Month < 4).Sum(x=>x.Item2);
```

Comment:
was nice idea to split into variables with correct naming, much more clear to understand